addons/web_editor/static/src/xml/editor.xml

```diff
@@ -206,6 +208,13 @@
             <div class="btn-group only_fa">
                 <div id="fa-spin" title="Toggle icon spin" class="editor-ignore btn fa fa-play"></div>
             </div>
+
+            <div t-if="props.showHistory" id="history" class="btn-group">
+                <div id="undo" data-call="undo" class="btn fa fa-undo"/>
+                <div id="redo" data-call="redo" class="btn fa fa-repeat"/>
+            </div>
+
+            <t t-slot="default"/>
```

Comment:
Oh we have slots now ? What are we slotting here ? Sounds very mysterious from here without any comment or anything... 🧙

Reply:
If I recall correctly, I didn't see the slot before so I thought it was something new that was added in this PR. If that is the case, then I feel like a comment would be useful. We can discuss it in 3D because I feel like we're not on the same page so I feel like I might be missing some piece of context here.